doc/python/imshow.md

```diff
@@ -110,6 +110,34 @@ fig.update_layout(coloraxis_showscale=False)
 fig.show()
 ```
 
+### Display an xarray image with px.imshow
+
+[xarrays](http://xarray.pydata.org/en/stable/) are labeled arrays (with labeled axes and coordinates). If you pass an xarray image to `px.imshow`, its axes labels and coordinates will be used for ticks. (If you don't want this behavior, just pass `img.values` which is a NumPy array if `img` is an xarray).
+
+```python
+import plotly.express as px
+import xarray as xr
+# Load xarray from dataset included in the xarray tutorial
+# We remove 273.5 to display Celsius degrees instead of Kelvin degrees
+airtemps = xr.tutorial.open_dataset('air_temperature').air.isel(lon=20) - 273.5
```

Comment:
would be cleaner to do `xr.tutorial.open_dataset('air_temperature').air.sel(lon=250.0)` no? that way we're doing a data-space selection (lon == 250.0 rather than 20th lon value) and we're not subtracting anything, so the underlying `long_name` is preserved... If we don't subtract, the attrs stick around and look like this:

```
<xarray.DataArray 'air' (time: 2920, lat: 25)>
array([[246.5    , 242.     , 240.39   , ..., 295.5    , 297.     , 298.29   ],
       [244.     , 241.59999, 240.59999, ..., 296.     , 297.9    , 299.19998],
       [242.2    , 239.59999, 239.89   , ..., 296.79   , 297.9    , 298.69998],
       ...,
       [246.68999, 246.89   , 248.89   , ..., 296.29   , 297.99   , 298.88998],
       [244.39   , 244.59   , 245.59   , ..., 296.99   , 298.19   , 299.49   ],
       [242.79   , 244.39   , 246.09   , ..., 296.09   , 297.99   , 299.19   ]],
      dtype=float32)
Coordinates:
  * lat      (lat) float32 75.0 72.5 70.0 67.5 65.0 ... 25.0 22.5 20.0 17.5 15.0
    lon      float32 250.0
  * time     (time) datetime64[ns] 2013-01-01 ... 2014-12-31T18:00:00
Attributes:
    long_name:     4xDaily Air temperature at sigma level 995
    units:         degK
    precision:     2
    GRIB_id:       11
    GRIB_name:     TMP
    var_desc:      Air temperature
    dataset:       NMC Reanalysis
    level_desc:    Surface
    statistic:     Individual Obs
    parent_stat:   Other
    actual_range:  [185.16 322.1 ]
```

Reply:
not sure which of these is "standard" for us to pick up on for additional labelling... the `actual_range` looks juicy for color_range ?

Reply:
ah in the xarray docs they do this:

```
# Convert to celsius
In [7]: air = airtemps.air - 273.15

# copy attributes to get nice figure labels and change Kelvin to Celsius
In [8]: air.attrs = airtemps.air.attrs

In [9]: air.attrs['units'] = 'deg C'
```